internal/rpc/jsonrpc/methods.go

```diff
@@ -2220,12 +2220,7 @@ func (s *Server) getMultisigOutInfo(ctx context.Context, icmd interface{}) (inte
 	}
 
 	// Get the list of pubkeys required to sign.
-	_, pubkeyAddrs, _, err := txscript.ExtractPkScriptAddrs(
-		0, p2shOutput.RedeemScript,
-		w.ChainParams(), true) // Yes treasury
-	if err != nil {
-		return nil, err
-	}
+	_, pubkeyAddrs := stdscript.ExtractAddrs(0 /* TODO */, p2shOutput.RedeemScript, w.ChainParams())
```

Comment:
i do wonder if we should create an unexported constant for these assumed script versions of the previous output scripts, so removing it will show all of the undefined reference errors

Reply:
`const scriptVersionAssumed = 0` would better than `0 /* TODO*/` in that regard, I agree